src/goto-programs/remove_returns.cpp

```diff
@@ -74,6 +74,7 @@ remove_returnst::get_or_create_return_value_symbol(const irep_idt &function_id)
 
   auxiliary_symbolt new_symbol;
   new_symbol.is_static_lifetime = true;
+  new_symbol.is_auxiliary = true;
```

Comment:
This is redundant, because `new_symbol` already is an `auxiliary_symbolt` (and thus has this flag set to `true` by construction).